### PR TITLE
playlab: 8 new projectiles

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

File: apps/src/studio/skins.js [hunk 1]
```diff
@@ -1570,31 +1602,66 @@ function loadStudio(skin, assetUrl) {
...
     'blue_fireball',
     'purple_fireball',
     'red_fireball',
     'purple_hearts',
     'red_hearts',
-    'yellow_hearts',
+    'yellow_hearts'
   ];
+
+  skin.specialProjectileProperties = {
```
Comment (Hamms) [inline apps/src/studio/skins.js:1627]: Is this being used anywhere?

**Comment by Hamms** [comment]
This LGTM, but I don't know enough about playlab to be able to identify anything that could be wrong.

However, this does seem like a silly amount of work necessary to do something as simple as add new assets. How difficult would it be to clean up our skins so as to make this simpler?

File: apps/src/studio/skins.js (hunk 1 above)
Comment (breville) [inline apps/src/studio/skins.js:1627]: Yes, this is consumed by studio.js (https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/studio/studio.js#L4895) and lets us override this skin's default number of 8 frames per projectile for these new animations.

**Comment by breville** [comment]
Completely agreed.. I was surprised it took so much to get these in.  Certainly it feels ripe for a refactor so that we just list the new projectiles in one place, but it would take somebody focused on this as a mini-project in its own right.

**Pull request merged**